.travis.yml

```diff
@@ -5,122 +5,131 @@ sudo: false
-    - if [ $TASK_TESTS == 1 ] && [ $TASK_TESTS_COVERAGE == 1 ]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover build/logs/clover.xml; fi
-
-    - if [ $TASK_SCA == 1 ]; then git checkout . -q; fi
-
-    - if [ $TASK_CS == 1 ]; then php php-cs-fixer --diff --dry-run -v fix; fi
-
-after_success:
-    - if [ $TASK_TESTS_COVERAGE == 1 ]; then php vendor/bin/coveralls -v; fi
-
-before_deploy:
-    # install box2
-    - curl -LSs http://box-project.github.io/box2/installer.php | php
-    - php box.phar --version
-
-    # ensure that deps will work on lowest supported PHP version
-    - composer config platform.php 2> /dev/null || composer config platform.php 5.3.6
-
-    # require suggested packages
-    - composer require --no-update symfony/polyfill-mbstring
-
-    # update deps to highest possible for lowest supported PHP version
-    - composer update $DEFAULT_COMPOSER_FLAGS --no-dev --prefer-stable
-
-    - composer info -D | sort
-
-    # build phar file
-    - php -d phar.readonly=false box.phar build
-
-deploy:
-    provider: releases
-    api_key:
-        secure: K9NKi7X1OPz898fxtVc1RfWrSI+4hTFFYOik932wTz1jC4dQJ64Khh1LV9frA1+JiDS3+R6TvmQtpzbkX3y4L75UrSnP1ADH5wfMYIVmydG3ZjTMo8SWQWHmRMh3ORAKTMMpjl4Q7EkRkLp6RncKe+FAFPP5mgv55mtIMaE4qUk=
-    file: php-cs-fixer.phar
-    skip_cleanup: true
-    on:
-        repo: FriendsOfPHP/PHP-CS-Fixer
-        tags: true
-        condition: $DEPLOY = yes
-
-after_deploy:
-    - ./dev-tools/trigger-website.sh ${TRAVIS_TOKEN} ${TRAVIS_TAG}
+jobs:
+    include:
+        -
+            stage: Static Code Analysis
+            php: 7.1
+            env: COMPOSER_FLAGS="--no-dev --prefer-stable"
+            install:
+                - travis_retry composer update -d dev-tools $DEFAULT_COMPOSER_FLAGS
+                - composer info -d dev-tools -D | sort
+
+                - travis_retry composer update $DEFAULT_COMPOSER_FLAGS $COMPOSER_FLAGS
+                - composer info -D | sort
+            script:
+                # @TODO remove at 3.0
+                - git archive -o /dev/null HEAD -v 2>&1 | grep tests | grep \.php | grep -v tests/Test/Assert/AssertTokensTrait.php | grep -v tests/Test/AbstractFixerTestCase.php | grep -v tests/Test/AbstractIntegrationTestCase.php | grep -v tests/Test/IntegrationCase.php | grep -v tests/Test/IntegrationCaseFactory.php && (echo "UNKNOWN FILES DETECTED" && travis_terminate 1) || echo "NO UNKNOWN FILES"
+
+                - ./check_trailing_spaces.sh || travis_terminate 1
+                - ./dev-tools/vendor/bin/composer-require-checker check composer.json --config-file=.composer-require-checker.json
+
+
+        - &STANDARD_TEST_JOB
+            stage: Test
+            php: 7.0
+            install:
+                # Composer: enforce given Symfony components version
+                - 'if [ "$SYMFONY_VERSION" != "" ]; then sed -i "s/\"symfony\/\([^\"]*\)\": \"^2[^\"]*\"/\"symfony\/\1\": \"$SYMFONY_VERSION\"/g" composer.json; fi'
```

Comment:
Where is `$SYMFONY_VERSION` defined?

Reply:
restored, it was on 2.2 line, now it's also on this PR